change BUS to SUBSYSTEMS in udev rules

There is one missing change in nut-usbups.rules.in: BUS -> SUBSYSTEMS. 
Previous change (rev2071) missed this. Now, the latest udev started to complain:
... udevd[31526]: BUS= will be removed in a future udev version ... 
Information about udev change can be found in udev repo:
http://git.kernel.org/?p=linux/hotplug/udev.git;a=commit;h=95776dc6ec174f47fa4dd4d8abf5d457223e5dd4

Index: nut/scripts/udev/nut-usbups.rules.in
===================================================================
--- nut/scripts/udev/nut-usbups.rules.in	(revision 2405)
+++ new/scripts/udev/nut-usbups.rules.in	(working copy)
@@ -3,7 +3,7 @@
 ACTION!="add", GOTO="nut-usbups_rules_end"
 SUBSYSTEM=="usb_device", GOTO="nut-usbups_rules_real"
 SUBSYSTEM=="usb", GOTO="nut-usbups_rules_real"
-BUS!="usb", GOTO="nut-usbups_rules_end"
+SUBSYSTEMS!="usb", GOTO="nut-usbups_rules_end"
 
 LABEL="nut-usbups_rules_real"
 #  Krauler UP-M500VA  - blazer_usb
